.github/actions/spelling/allow/occ.txt

```diff
@@ -9,6 +9,7 @@ CLA
 contentpart
 CVC
 datetimefield
```

Comment:
I discovered a bug: When you fail the first try of payment (e.g. I tried using the test card with insufficient funds), then the next attempt will fail with the same error, even if I use the successful card: 
![gifff](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/80963259/03fbb88a-e93a-4562-b73c-c5d2bf207504)